Sources/hostmgr-helper/VM List/VMListItem.swift

```diff
@@ -8,15 +8,15 @@ struct VMListItem: View {
     var body: some View {
         switch slot.status {
         case .empty: EmptyVMListItem(slot: slot)
-        case .starting(let launchConfiguration):
+        case .starting(let launchConfiguration, _):
             PendingVMListItem(
                 launchConfiguration: launchConfiguration,
                 slot: slot
             )
-        case .running(let launchConfiguration, let ipAddress):
+        case .running(let mvm):
             RunningVMListItem(
-                launchConfiguration: launchConfiguration,
-                ipAddress: ipAddress,
+                launchConfiguration: mvm.config,
+                ipAddress: mvm.ip,
```

Comment:
What do you think of using a more verbose but also more explicit `managedVirtualMachine` here? Or even just `virtualMachine`?

`mvm` makes me think of MVVM more than VM. That might just be me, but there might be something to be said for keeping this codebase where we work infrequently as clear to read as possible.

Reply:
> mvm makes me think of MVVM
Had that same thought. 😅 `managedVirtualMachine` seemed too long, and `vm` seemed to be a poor representation. I'll update this to be clearer.

Reply:
Handled in 30aaed3da16ca22e6b0d5efd1b744c6845b6ba33.